client/pages/login.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+  <AuthForm v-slot="{ invalid }" @submit="signin">
+    <AuthAlert :errors="errors">
+      Error while signing in, please fix the following errors:
+    </AuthAlert>
+    <InputField
+      id="email"
+      class="w-full"
+      rules="required|email"
+      field-name="Email"
+      field-type="text"
+    />
+    <InputField
+      id="password"
+      class="w-full"
+      rules="required"
+      field-name="Password"
+      field-type="password"
+    />
+    <NuxtLink
+      to="/forgot-password"
+      class="self-end text-xs font-semibold text-white underline underline-offset-2"
+    >
+      forgot password?
+    </NuxtLink>
+    <AuthSubmit :disabled="invalid">Login</AuthSubmit>
+    <p class="place-self-center">
+      Don't have an account?
+      <NuxtLink
+        to="/sign-up"
+        class="text-blue font-semibold underline underline-offset-2"
+      >
+        Sign up!
+      </NuxtLink>
+    </p>
+  </AuthForm>
+</template>
+
+<script>
+export default {
+  layout: 'auth',
+  data() {
+    return {
+      errors: {},
+    };
+  },
+  head: {
+    title: 'Login',
+  },
+  methods: {
+    async signin(e) {
+      // Get the values from the form
+      const elements = e.target.elements;
+      const data = {
+        email: elements.Email.value,
```

Comment:
An alternative way to do this would be to store a `email` and `password` in the `data` section and just `v-model` to your input values like this:
```
<InputField
    id="email"
    class="w-full"
    rules="required|email"
    field-name="Email"
    field-type="text"
    v-model="email"
/>
<InputField
    id="password"
    class="w-full"
    rules="required"
    field-name="Password"
    field-type="password"
    v-model="password"
/>
...
data() {
    return {
        email: "",
        password: "",
    }
}
```

Reply:
It would save you trying to pull the values out of `e.target.elements`

Reply:
I thought of this option first when refactoring. Doing this would introduce breaking changes as `InputField` on main can't take the value as a prop.